src/utils/geom.js

```diff
@@ -7,6 +7,29 @@ export const MULTI_LINESTRING = 5;
 export const MULTI_POLYGON = 6;
 export const GEOMETRY_COLLECTION = 7;
 
+export const getShapeFromGeomType = geomType => {
+  switch (geomType) {
+    case POINT:
+    case MULTI_POINT:
+      return 'circle';
+    case LINESTRING:
+    case MULTI_LINESTRING:
+      return 'line';
+    default:
+      return 'fill';
+  }
+};
+
+export const getLayerStyleDefaultValue = (color, geomType) => {
+  const type = getShapeFromGeomType(geomType);
+  return {
+    type,
+    paint: {
+      [`${type}-color`]: color,
+    },
+  };
+};
+
```

Comment:
Just a suggestion, not mandatory:

```jsx
export const getLayerStyleDefaultValue_ = (
  color,
  geomType,
  type = getShapeFromGeomType(geomType),
) => ({
  type,
  paint: {
    [`${type}-color`]: color,
  },
});
```

Reply:
I'm not sure about this one.
The `geomType` argument is only useful for getting the shape, we don't need it for anything else.
If we set the third parameters `type`, the `geomType` in the middle seems confusing.

Reply:
Could also be done with object as function arg:

```jsx
export const getLayerStyleDefaultValue = ({
  color,
  geomType,
  type = getShapeFromGeomType(geomType),
}) => ({
  type,
  paint: {
    [`${type}-color`]: color,
  },
});
```